blog/2022-maintainer-summit.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Maintainer Summit 2022 Recap
+date: 2022-10-13T00:00:00.000Z
+authors:
+  - name: erickzhao
+    url: 'https://github.com/erickzhao'
+    image_url: 'https://github.com/erickzhao.png?size=96'
+slug: maintainer-summit-2022-recap
+---
+
+# Maintainer Summit 2022 Recap
+
+Last month, Electron’s maintainer group met up in Vancouver, Canada to discuss the direction of the
+project for 2023 and beyond. Over four days in a conference room, core maintainers and invited
+collaborators discussed new initiatives, maintenance pain points, and general project health.
+
+<figure>
+  <img src="/assets/img/2022-maintainer-summit.jpg"/>
+  <figcaption align="center">
+    Group Photo! Taken by <a href="https://github.com/groundwater">@groundwater</a>.
+  </figcaption>
+</figure>
+
+Going forward, the team will still be fully dedicated to releasing regular and rapid
+Chromium upgrades, fixing bugs, and making Electron more secure and performant for everyone.
+We also have a few exciting projects in the works we would love to share with
+the community!
+
+## Transformative new APIs
+
+Major API proposals in the Electron project that require consensus go through a Request for Comments
+(RFC) process, which gets reviewed by members of our API Working Group.
+
+This year, we have driven forward two major proposals that have the potential to unlock a new
+dimension of capabilities for Electron apps. These proposals are highly experimental, but here’s a
+sneak peek of what to expect! 
+
+### New native addon enhancements (C APIs)
+
+This proposal outlines a new layer of Electron C APIs that will allow app developers to write their
+own Native Node Addons that interface with Electron’s internal resources, similar to Node’s
+own Node-API.
```

Comment:
Do we want to link to the spec doc, so people can comment?

```suggestion
own Node-API. More information about the proposed new API [can be found here](https://github.com/electron/governance/blob/main/wg-api/spec-documents/electron-c-apis.md).
```

This may also be a separate, larger question - do we have a place where we currently post/gather comments for RFCs like this one? I know we were curious to get example use cases from people at Summit - there may be a good place for us to get similar use cases here.